tests/basic-http.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/env python3
+import argparse
+import http.server
+import os
+import shutil
+import ssl
+import subprocess
+import tempfile
+import threading
+import time
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--naive', required=True)
+parser.add_argument('--rootfs')
+parser.add_argument('--target_cpu')
+argv = parser.parse_args()
+
+if argv.rootfs:
+    try:
+        os.remove(os.path.join(argv.rootfs, 'naive'))
+    except OSError:
+        pass
+
+HTTP_SERVER_HOSTNAME = '127.0.0.1'
+HTTP_SERVER_PORT = 60080
+httpd = http.server.HTTPServer(
+    (HTTP_SERVER_HOSTNAME, HTTP_SERVER_PORT), http.server.SimpleHTTPRequestHandler)
+httpd.timeout = 1
+httpd.allow_reuse_address = True
```

Comment:
Should parametrize basic.py to support http test instead of duplicating the code.

Reply:
Done.